pom.xml

```diff
@@ -9,15 +9,14 @@
     <relativePath />
   </parent>
   <artifactId>tuleap-git-branch-source</artifactId>
-  <version>1.0.8-SNAPSHOT</version>
+  <version>1.0.9-SNAPSHOT</version>
```

Comment:
This is modified by the release process. It should not be modified outside of it.